File: CS-Examples/02_Data/DataExport.cs
using System;
using System.Data.OleDb;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;
using System.Data;

using Spire.Xls;

namespace DataExport
{
	public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
		private void btnRun_Click(object sender, System.EventArgs e)
		{
            // Create a new workbook object to work with Excel files
            Workbook workbook = new Workbook();

            // Load file
            workbook.LoadFromFile(@"..\..\..\..\..\..\Data\DataExport.xlsx");

            // Get first sheet
            Worksheet sheet = workbook.Worksheets[0];

            // Export data 
            this.dataGrid1.DataSource = sheet.ExportDataTable();

            // Dispose of the workbook object to free up resources
            workbook.Dispose();
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }



	}
}

File: CS-Examples/02_Data/DataExport.vb
Imports System.Data.OleDb
Imports System.Collections
Imports System.ComponentModel

Imports Spire.Xls

Namespace DataExport
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            ' Create a new Workbook object
            Dim workbook As New Workbook()

            ' Load an existing Excel file into the workbook
            workbook.LoadFromFile("..\..\..\..\..\..\Data\DataExport.xlsx")

            ' Initialize a Worksheet object by getting the first worksheet from the workbook
            Dim sheet As Worksheet = workbook.Worksheets(0)

            ' Set the data source of a DataGrid (assuming "dataGrid1" is a DataGrid control) to the exported DataTable from the worksheet
            Me.dataGrid1.DataSource = sheet.ExportDataTable()

            ' Release the resources used by the workbook
            workbook.Dispose()

        End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub



	End Class
End Namespace
